rector.php

```diff
@@ -0,0 +1,31 @@
+<?php
+declare(strict_types=1);
+
+use Rector\Core\Configuration\Option;
+use Rector\Php74\Rector\Property\TypedPropertyRector;
+use Rector\Set\ValueObject\LevelSetList;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    // get parameters
+    $parameters = $containerConfigurator->parameters();
+    $parameters->set(Option::PATHS, [
+        __DIR__ . '/src'
+    ]);
+    $parameters->set(Option::SKIP, [
+        \Rector\Php71\Rector\ClassConst\PublicConstantVisibilityRector::class,
+    ]);
+
+    // Define what rule sets will be applied
+    $containerConfigurator->import(LevelSetList::UP_TO_PHP_72);
+    $containerConfigurator->import(LevelSetList::UP_TO_PHP_81);
```

Comment:
Are both these here because UP_TO_PHP_81 does not include UP_TO_PHP_72?

Reply:
After knowing what I know now, I think I could remove `UP_TO_PHP_72`, as long as the following line is included.
```
$parameters->set(Option::PHP_VERSION_FEATURES, \Rector\Core\ValueObject\PhpVersion::PHP_72);
```
Although, I'm not entirely sure, and I'm ok leaving it in, since it doesn't hurt anything.  I'll plan to remove it in the future.